packages/squidex/schema/schemas/research-outputs.json

```
@@ -213,6 +215,23 @@
           "isRequiredOnPublish": false,
           "isHalfWidth": false
         }
+      },
+      {
+        "name": "lastUpdatedPartial",
+        "isHidden": false,
+        "isLocked": false,
+        "isDisabled": false,
```

```suggestion
        "isDisabled": true,
```

Just missed this bit. The field should not be editable, right?